### ISSUE TITLE
Add FTP protocol for external service entry

### ISSUE THREAD

**Issue by courcelm** [istio/istio#15776]
**Describe the feature request**

Add the FTP protocol to Service entry to allow file download from external FTP. FTP start the connection with TCP and then switch to UDP. I did find a way to allow two protocols on the same port.

**Describe alternatives you've considered**

**Affected product area (please put an X in all that apply)**

[ ] Configuration Infrastructure
[ ] Docs
[ ] Installation
[ x] Networking
[ ] Performance and Scalability
[ ] Policies and Telemetry
[ ] Security
[ ] Test and Release
[ ] User Experience
[ ] Developer Infrastructure

**Additional context**

**Comment by howardjohn** [comment]
We don't yet support UDP so we are blocked by https://github.com/istio/istio/issues/1430

**Comment by howardjohn** [comment]
Closing as duplicate of https://github.com/istio/istio/issues/15972
